tests/common/sai_validation/gnmi_ext_pb2.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
>Generated [](http://example.com/codeflow?start=2&length=9)

If this file is auto generated, no need to check-in git.

Reply:
There is a small modification to these files to fix the import path. I think generating these files would require some sort of setup or build step in sonic management container cloning gnmi repository and running protoc. It may be easier to have them in as the python version on the sonic management container doesn't change often.

Reply:
Below are my worry:
1. how to regenerate this file
2. when to regenerate this file
3. how to fix the import path

We need ensure all the knowledge captured in code, or at least comments.

Reply:
@qiluo-msft I have modified to code to generate the gRPC stub code at runtime.